tests/Utilities/utils.py

```diff
@@ -0,0 +1,6 @@
+''' Adds multiple foods into our food storage. '''
```

Comment:
What is this file for?

Reply:
Yes! These methods will probably be used frequently in our test files, so I decided to create a `utils` files for it. I've seen this done at some companies I've worked at.